frontend/src/components/dashboard/page_views/ButtonComponent.vue

```diff
@@ -0,0 +1,126 @@
+<template>
```

Comment:
don't copy/paste, create 1 common component, that will work with just props, use it here and in visitors page